benchmark/templates/rgw_bench_test.conf.j2

```diff
@@ -0,0 +1,11 @@
+[bench]
+auth=http://{{ internal_lb_vip_address }}:8080/auth/v1.0
+user=testuser:swift
+key=swiftsecret
+concurrency={{ bench_rgw_concurrency | default(15) }} 
```

Comment:
Fixed 
I've changed the defaults to be something you would use on a real world customer
concurrency  = 50
object size = 524288 . (512k)
number of objects = 250000
number of gets = 500000

This will write 128GB worth of data into the cluster (Not taking into account the storage strategy) 

We need to be sure and set AIO appropriate values in the tests/test-vars.yml file or it will blow up the virtual machine